src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ImageTests/LoadFromBytes.xaml.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Uno.UI.Samples.Controls;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Imaging;
+
+namespace Uno.UI.Samples.UITests.ImageTestsControl
+{
+	[SampleControlInfo("Image", "LoadFromBytes")]
+	public sealed partial class LoadFromBytes : UserControl
+	{
+		private readonly string _imageUrl = "https://www.spriters-resource.com/resources/sheet_icons/30/32521.png";
+
+		public LoadFromBytes()
+		{
+			InitializeComponent();
+			Loaded += OnLoaded;
+		}
+
+		private async void OnLoaded(object sender, RoutedEventArgs e)
+		{
+			var client = new HttpClient(new CorsByPassHandler());
+
+			var result = await client.GetAsync(_imageUrl);
+			var stream = await result.Content.ReadAsStreamAsync();
+
+
+			var bitmap = new BitmapImage();
+#if !NETFX_CORE
+			await bitmap.SetSourceAsync(stream);
+#else
+				var memStream = new MemoryStream();
+				await stream.CopyToAsync(memStream);
+				memStream.Position = 0;
+				using (memStream)
+				{
+					await bitmap.SetSourceAsync(memStream.AsRandomAccessStream());
+				}
+#endif
+			MyImage.Source = bitmap;
+
+		}
+
+		public class CorsByPassHandler : DelegatingHandler
+		{
+			public CorsByPassHandler()
+			{
+				InnerHandler = new HttpClientHandler();
+			}
+
+			protected override Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
+			{
+				var builder = new UriBuilder(request.RequestUri)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'request' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5223146)